api/src/test/resources/config/entity/ConditionOccurrence.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "condition_occurrence",
+  "idAttribute": "id",
+  "attributes": [
+    { "type": "SIMPLE", "name": "id", "dataType": "INT64" },
+    { "type": "SIMPLE", "name": "person_id", "dataType": "INT64" },
+    { "type": "KEY_AND_DISPLAY", "name": "condition_id", "dataType": "INT64" },
+    { "type": "SIMPLE", "name": "condition_start_date", "dataType": "STRING" },
+    { "type": "SIMPLE", "name": "condition_end_date", "dataType": "STRING" },
+    { "type": "SIMPLE", "name": "stop_reason", "dataType": "STRING" },
+    { "type": "SIMPLE", "name": "visit_occurrence_id", "dataType": "INT64" },
+    { "type": "SIMPLE", "name": "condition_source_value", "dataType": "STRING" },
+    { "type": "SIMPLE", "name": "condition_source_concept_id", "dataType": "INT64" } ],
+
+  "sourceDataMapping": {
+    "dataPointer": "omop_dataset",
+    "tablePointer": { "table": "condition_occurrence" },
+    "attributeMappings": {
+      "id": { "value": { "column": "condition_occurrence_id" } },
```

Comment:
Why is `column` needed here? Are there other valid options?

Reply:
Yes, it could be a foreign-keyed column
```
{
          "column": "vocabulary_id",
          "foreignTable": "vocabulary",
          "foreignKey": "vocabulary_id",
          "foreignColumn": "vocabulary_name" }
```
or include a SQL function wrapper
```
{
          "column": "standard_concept",
          "sqlFunctionWrapper": "(CASE WHEN ${fieldSql} IS NULL THEN 'Source' WHEN ${fieldSql} = 'S' THEN 'Standard' ELSE 'Unknown' END)" }
```

Reply:
Okay, that makes sense.